Library/Formula/rethinkdb.rb

```diff
@@ -75,6 +73,11 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    shell_output("#{bin}/rethinkdb create -d test")
```

Comment:
CI test failed when I use `system`. It appears that in verbose mode, `system` call would hang out indefinitely. Log is in below:

```
info: Our machine ID: 7581c949-a486-4c78-b802-43b01cebeb08
Previous message may not have been written to the log file (cannot flush stderr: Operation not supported).
info: Created directory 'test' and a metadata file inside it.
Previous message may not have been written to the log file (cannot flush stderr: Operation not supported).
```

I think the relate code is [here](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/formula.rb#L659-L675). The `system` function doesn't handle stderr well. @jacknagel